docs/core-plugins/e2e-cypress.md

```diff
@@ -37,6 +37,8 @@ Cypress offers a rich interactive interface for running E2E tests, but currently
 
 We've pre-configured Cypress to place most of the e2e testing related files under `<projectRoot>/tests/e2e`. You can also check out [how to configure Cypress via `cypress.json`](https://docs.cypress.io/guides/references/configuration.html#Options).
 
+Defining `baseUrl` in `cypress.json` won't have any effect, the only way to change `baseUrl` is trough the CLI flag `--url`. Ex. `vue-cli-service test:e2e --url https://localhost:8080`. 
```

Comment:
It seems to be rather a bug than something, that should be mentioned in the documentation.

Why `baseUrl` in `cypress.json` won't have any effect?